app/medInria/medRegistrationWorkspace.cpp

```diff
@@ -132,25 +121,25 @@ bool medRegistrationWorkspace::isUsable()
 
 void medRegistrationWorkspace::updateFromMovingContainer()
 {
-    if(!d->selectorToolBox)
+    medRegistrationSelectorToolBox* toolbox = dynamic_cast<medRegistrationSelectorToolBox*>(selectorToolBox());
+    if(!toolbox)
```

Comment:
it's not easily readable, there is no space, and it would be better to put instead:
```c++
if(toolbox)
{
    blabla
}
```